.github/workflows/test.yml

```diff
@@ -9,8 +9,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: jtdor/build-deb-action@v1.0.0
-        with:
-          docker-image: debian:unstable-slim
-          buildpackage-opts: --build=binary --no-sign
+      # FIXME massive hack, just trying to see if this works
+      - name: build and test
+        run: |
+          set -ex
+          git fetch --all
+          git branch pristine-tar origin/pristine-tar
```

Comment:
Some Debian packaging teams use the `upstream` branch + upstream tags only - that is, setting `pristine-tar = False` in `debian/gbp.conf` or using `--git-no-pristine-tar`.  I'm still kind of partial to `pristine-tar`, and there is no need to change this.  I merely wanted to point it out because you'll see it in some packaging repos.

Also, I should have mentioned sooner that the branch names come from [DEP-14](https://dep-team.pages.debian.net/deps/dep14/), which is still in draft and offers a bit of leeway with respect to branch-naming, but technically, the `debian/master` branch should be either `debian/latest` or `debian/unstable` with the former being preferred.  And the 0.9.x packaging will live in `debian/experimental` for bit.

Reply:
>  I'm still kind of partial to pristine-tar, and there is no need to change this. I merely wanted to point it out because you'll see it in some packaging repos.

Right, I think I knew this at one point but good to get the reminder. 🙂 I do think I prefer using import-orig + tarballs so I know I'm getting the "true" release artifact if that makes sense but it's kind of a tomato/tomato thing really.

> Also, I should have mentioned sooner that the branch names come from DEP-14, 

You did mention this though must admit I haven't read it super closely, I just inferred the meaning of the new branches I guess.

> technically, the debian/master branch should be either debian/latest or debian/unstable with the former being preferred. And the 0.9.x packaging will live in debian/experimental for bit.

Got it, so when we `gbp import-orig` we want it to land in the experimental branch? Understood. I know there'd be ways to make it work but I like that there's kind of a norm/paved path for "make it possible to bring in a new version, make it easy to maintain the old one".

Reply:
For the general use case, `debian/latest` is appropriate.  We would use `debian/experimental` only if we knew were going to be part of an SO-name bump/transition, or were doing something else interesting enough to not be suitable for unstable. I realize that capnp might be a bit different in this regard - maybe it is frequently going to introduce SO-name bumps, but typically packages land in unstable.